dev/ory/oathkeeper_rules.yaml

```diff
@@ -42,15 +42,15 @@
         claims: '{"sub": "{{ print .Subject }}"}'
 - id: admin-backend
   upstream:
-    url: "http://e2e-tests:4001"
+    url: "http://e2e-tests:4011"
     strip_path: /admin
   match:
     url: "<(http|https)>://<.*><[0-9]+>/admin<.*>"
     methods: ["GET", "POST", "OPTIONS"]
   authenticators:
     - handler: bearer_token
       config:
-        check_session_url: "http://e2e-tests:4001/auth/validatetoken"
+        check_session_url: "http://e2e-tests:4011/auth/validatetoken"
```

Comment:
@krtk6160 does this look good? 4011=>4001 i changed it to follow how we do it for the main backend and 4012=>4002